cves/CVE-2010-3082.yml

```diff
@@ -124,10 +128,13 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability appears to have been discovered by Jeff Balogh of Mozilla. There is no information on when or how
+    exactly it was discovered. I have looked at all provided links in nvd.nist.gov, as well as numerous searches involving
+    Jeff or the vulnerability.
```

Comment:
It's great that you have looked everywhere. In this case, just put down the link you find that says it's been discovered by Jeff. Or a bug report. That would be fine.

Reply:
This is the bug report that credits Jeff. https://bugzilla.redhat.com/show_bug.cgi?id=632239